src/lib/y2partitioner/widgets/md_description.rb

```diff
@@ -57,12 +57,22 @@ def raid_description
       # @return [Array<String>]
       def raid_attributes
         [
+          raid_active,
           raid_type,
           raid_chunk_size,
           raid_parity
         ]
       end
 
+      # Information about MD RAID being (in)active
+      #
+      # @return [String]
+      def raid_active
+        # TRANSLATORS: RAID being active (assembled), where %s is replaced by
+        # 'Yes' when the device is active or by 'No' otherwise
+        format(_("Active: %s"), blk_device.active? ? _("Yes") : _("No"))
```

Comment:
Maybe it's more translation-friendly to have the strings `"Active: Yes"` and `"Active: No"`

Reply:
Well, mayybe, but this way it is consistent with "Encrypted: Yes/No" which we already have.

Reply:
Ah. Ok.